rust-runtime/aws-smithy-types/src/date_time/format.rs

```diff
@@ -405,12 +405,6 @@ pub(crate) mod rfc3339 {
     // Timezones not supported:
     // Not OK: 1985-04-12T23:20:50-02:00
     pub(crate) fn parse(s: &str) -> Result<DateTime, DateTimeParseError> {
-        if !matches!(s.chars().last(), Some('Z')) {
```

Comment:
Was removing this intentional?

Reply:
yeah—Smithy changed their spec and now allows timezones